### ISSUE TITLE
No Text Editor Found

### ISSUE THREAD

**Issue by nhocki** [topfunky/PeepOpen-Issues#65]
When I open Peepopen I get an empty box and when I press Enter I get a warning saying "No text editor found" even though I have TextMate open.

In the Console, I get an "undefined method [] for nil:NilClass". I've installed the new version of peeopen and re-installed it and still get the same problem.

Thanks,

**Comment by topfunky** [comment]
Try opening PeepOpen preferences and selecting an Editor in the Editor tab. Is anything listed?

**Comment by nhocki** [comment]
Yeah. It says TextMate. The wired thing is that I open another folder and peepopen works. When I open another it won't work... I could send you screenshots and the console errors if you want... It may be useful for you to find out what's going on...

Thanks,

**Comment by conceptric** [comment]
Downloaded PeepOpen today and got the same result on my MacBook Pro 3.1, OS-X 10.6.4, TextMate 1.5.8 (1509).

TextMate plugin installed OK, Command-T any file I know to be in the project (even the one currently open) and "No text editor found" appears when I hit Enter. 

I've tried reselecting TextMate from the preferences (it was already there), reinstalling the plugin, reinstalling PeepOpen, but it's not helped.

James

**Comment by topfunky** [comment]
For debugging, could you send the preferences from ~/Library/Preferences/com.topfunky.PeepOpen.plist to boss@topfunky.com

I've had this happen sporadically, but haven't been able to recreate a situation where it happens every time.

**Comment by nhocki** [comment]
I can never use peepopen when I have the Community Engine plugin installed (on a rails app). If I open another project that does not use CE it works =/

You can get CE here http://github.com/bborn/communityengine
